src/transform.js

```diff
@@ -143,7 +143,7 @@ export function transformCommonjs ( parse, code, id, isEntry, ignoreGlobal, igno
 			programDepth += 1;
 
 			if ( node.scope ) scope = node.scope;
-			if ( /^Function/.test( node.type ) ) lexicalDepth += 1;
+			if ( /Function/.test( node.type ) ) lexicalDepth += 1;
```

Comment:
Very minor improvement: If you extract the RegExp into a constant (outside the transform function), then performance will be slightly better as it only needs to be compiled once and we can reuse it for both occurrences .